docs/content/guides.mdx

```diff
@@ -76,6 +76,24 @@ Learn to apply [Dagster concepts](/concepts) to your work, explore experimental
 
 ### Experimental features
 
+#### Dagster Pipes
+
+- [Dagster Pipes](/guides/dagster-pipes) - A high-level look at Dagster Pipes, a toolkit for building integrations between Dagster and external execution environments
+
+- [Dagster Pipes tutorial](/guides/dagster-pipes/subprocess) - Get started using Dagster Pipes with this tutorial, where you'll use a subprocess to execute code in an external environment
```

Comment:
very tactically, in order to get the build green, can you comment out this line? and i can uncomment it in my subprocess PR where the page is created.